app/views/editor/verifier/i18nVerifierView.coffee

```diff
@@ -0,0 +1,156 @@
+RootComponent = require 'views/core/RootComponent'
+Problem = require 'views/play/level/tome/Problem'
+locale = require 'locale/locale'
+
+I18nVerifierComponent = Vue.extend
+  template: require('templates/editor/verifier/i18n-verifier-view')()
+  data: ->
+    allLocales: Object.keys(_.omit(locale, 'update', 'installVueI18n')).concat('rot13')
+    language: 'en'
+    levelSlug: location.href.match('/editor/i18n-verifier/(.*)')?[1]
+    startDay: '2017-05-01'
+    endDay: '2017-07-30'
+    partialThreshold: 1
+    completeThreshold: 99
+    countThreshold: 0
+    totalCount: 0
+    messageOrHint: 'message'
+    me: me
+    serverConfig: serverConfig
+    problemsByLevel: {}
+    regexes: []
+    otherRegexes: []
+    displayMode: 'human-readable'
+    showCampaigns: false
+    showLevels: false
+    showTranslated: true
+    showUntranslated: true
+    campaigns: []
+    selectedCampaign: null
+    selectedLevelSlugs: [_.last(location.href.split('/'))]
+    loading: true
+  computed:
+    exportList: ->
+      _(@problems).filter((p) =>
+        p[@messageOrHint].length > 0 and\
+          @percentDifference(p) < @completeThreshold and\
+          (p.count / @totalCount) >= (@countThreshold / 100))
+      .uniq((p) -> p.trimmed)
+      .value()
+    problems: ->
+      _.sortBy(_.flatten(Object.values(@problemsByLevel), true), (p) -> -p.count)
+  created: ->
+    i18n.setLng(@language)
+    @loadCampaigns()
+    @loadLanguage(@language).then =>
+      @setupRegexes()
+      @getProblems(@levelSlug).then (newProblems) =>
+        @compareStrings(newProblems)
+        @loading = false
+  watch:
+    language: ->
+      @loadLanguage(@language).then =>
+        @setupRegexes()
+        @compareStrings(@problems)
+    selectedLevelSlugs: ->
+      @loading = true
+      promises = []
+      for slug in @selectedLevelSlugs
+        if not @problemsByLevel[slug]
+          promises.push @getProblems(slug)
+      Promise.all(promises).then (newProblems) =>
+        @loading = false
+        _.defer =>
+          @compareStrings(_.flatten(newProblems))
+    messageOrHint: ->
+      @compareStrings(@problems)
+  methods:
+    loadLanguage: (language) ->
+      new Promise (accept, reject) =>
+        loading = application.moduleLoader.loadLanguage(language)
+        if loading
+          application.moduleLoader.once 'load-complete', accept
+        else
+          accept()
```

Comment:
Perhaps change `loadLanguage` to return a promise by default? If it wouldn't take long to update the other places that function is used.